tasker/executors/sequential_executor.py

```
@@ -62,30 +62,49 @@ def execute_task(task, executor_instance):
+            success_result = True
+
+            # Store minimal results
+            executor_instance.store_task_result(task_id, {
+                'exit_code': exit_code,
+                'stdout': stdout,
+                'stderr': stderr,
+                'success': success_result
+            })
+
+            # Now handle the return
             if executor_instance.final_command == 'N/A': executor_instance.final_command = 'return'
```

Ruff (0.13.3)

Multiple statements on one line (colon)

(E701)